### ISSUE TITLE
Builds the wrong URL when origin uses git://

### ISSUE THREAD

**Issue by aelesbao** [ben-gibson/jetbrains-open-in-git-host#17]
When I have this setting in my remotes:
```
origin	git://github.com/aelesbao/repo (fetch)
origin	git@github.com:aelesbao/repo (push)
```
The built URL is `https://github/...` (without .com). If I change the remote URL to `git@github.com:aelesbao/repo` for both push and fetch, then it works.

**Comment by ben-gibson** [comment]
Thanks, I will take a look tonight. It doesn't look like I have accounted for that protocal in the [tests](https://github.com/ben-gibson/jetbrains-open-in-git-host/blob/master/tests/uk/co/ben_gibson/repositorymapper/RepositoryProvider/Repository/RepositoryTest.java#L52).

**Comment by ben-gibson** [comment]
This should be fixed in version 1.5.4 that has just been released. Could you download the update from your IDE and let me know?

**Comment by aelesbao** [comment]
Tested and it's working fine. Thanks!
